src/services/api/routes/v1/login.js

```diff
@@ -91,12 +89,21 @@ router.post('/', async (request, response) => {
 		});
 	}
 
-	const publicKey = fs.readFileSync(`${cryptoPath}/public.pem`);
-	const hmacSecret = fs.readFileSync(`${cryptoPath}/secret.key`);
+	let publicKey= await cache.getServicePublicKey('account');
+	if (publicKey === null) {
+		publicKey = await fs.readFile(`${cryptoPath}/public.pem`);
+		await cache.setServicePublicKey('account', publicKey);
+	}
```

Comment:
I'd recommend making the cache autoread if it doesn't exist in the cache.
that way you don't have this check everytime.
then its just `const key = await cache.getsomething('access/public.pem', 'account')`.